vignettes/applications.Rnw

```diff
@@ -314,6 +309,13 @@ gg.out <- ggplot()+
         hjust=1, x=169.5, y=5, label="Mean model", type="model"),
       data.table(
         hjust=1, x=169.4, y=spike.y, label="Predicted spikes", type="model"))))
+if(nrow(show.spikes)){
```

Comment:
this is needed now because gfpop is not returning any changepoints/segments. @vrunge can you please investigate?